src/main/shell-session/shell-session.ts

```diff
@@ -298,17 +294,16 @@ export abstract class ShellSession {
 
   protected async getCachedShellEnv() {
     const { id: clusterId } = this.cluster;
-    const { ShellType } = this;
 
-    let env = ShellSession.shellEnvs.get({ clusterId, ShellType });
+    let env = ShellSession.shellEnvs.get(clusterId);
```

Comment:
Changing the caching in a different PR?

Reply:
hmm, no, maybe I missed saving a file? will fix

Reply:
that file is now untouched by this PR, are you looking at all commits?

Reply:
I was not, I meant the change to only having temporal validity as the caching technique. Which could very well be its own PR.

Reply:
right, definitely its own PR. If we even need it? It doesn't impact the ShellEnvModifiers now